commands/tenants

```diff
@@ -7,13 +7,17 @@ help() {
 HELP MESSAGE:
     Usage: c8y bpl $(basename "$0") <command> --username <username> --password <password> --domain <domain-name>
     Commands:
-      create                    Create new tenant
-      help                      Show this help message and exit
+      create                                    Create new tenant
+      help                                      Show this help message and exit
     Arguments:
-      -u, --username  USERNAME  The username for the new user (required)
-      -p, --password  PASSWORD  The password for the new user ( Password must have at least 8 characters and no more than 32 and must contain only Latin characters) (required)
-      -d, --domain    DOMAIN    The domain for new sub-tenant (will be included ".latest.stage.c8y.io" automatically. Can contain  lowercase letters, digits, and hyphens; minimum 2 characters) (required)
-      -h, --help                Show this help message and exit
+      --adminName           ADMIN_USERNAME      The username for the new user (required)
+      --adminPass           ADMIN_PASSWORD      The password for the new user ( Password must have at least 8 characters and no more than 32 and must contain only Latin characters) (required)
+      --adminEmail          ADMIN_EMAIL         The email of the new user (required)
+      --name                NAME                The name for domain of new sub-tenant (will be included ".latest.stage.c8y.io" automatically. Can contain  lowercase letters, digits, and hyphens; minimum 2 characters) (required)
```

Comment:
- dont mention ".latest.stage.c8y.io" but replace with the more generic "c8y.instance.com"
- check for double whitespaces "  "